pkgs/applications/graphics/ahoviewer/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, pkgs, fetchurl, fetchFromGitHub, pkgconfig, libconfig, gtkmm, glibmm, libxml2, gstreamer, libsecret, curl, unrar, libzip, autoconf, automake }:
+with stdenv.lib;
+stdenv.mkDerivation rec {
+  name = "ahoviewer-1.4.6";
+  rev = "414cb91d66d96fab4b48593a7ef4d9ad461306aa";
+  src = fetchFromGitHub {
+    owner = "ahodesuka";
+    repo = "ahoviewer";
+    inherit rev;
+    sha256 = "081jgfmbwf2av0cn229cf4qyv6ha80ridymsgwq45124b78y2bmb";
+  };
+  buildInputs = [ pkgconfig glibmm libconfig gtkmm glibmm libxml2 gstreamer libsecret curl unrar libzip autoconf automake ];
+  preConfigure = ''
+    patchShebangs bootstrap
+    patchShebangs version.sh
+    ./bootstrap
+  '';
+  meta = {
+    homepage = "https://github.com/ahodesuka/ahoviewer";
+    description = "A GTK2 image viewer, manga reader, and booru browser";
+    license = "MIT";
```

Comment:
Would you mind adding yourself as maintainer? The maintainers list can be found in `lib/maintainers.nix`.